### PR TITLE
Limit operations blocking to Download phase of initial sync

### PR DESCRIPTION
**Pull request by anvmn**
https://github.com/Gizra/ihangane/issues/3068

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-08-15 12-50

**Comment by anvmn** [comment]
Peek 2023-08-15 12-57

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Pages/Utils.elm [hunk 1]
```diff
@@ -1246,3 +1250,38 @@ customPopup language showWarning actionLabel ( topMessage, bottomMessage, action
...
+    in
+    selectedHealthCenterSyncInfo
+        |> Maybe.map
+            (\syncInfo ->
+                case syncInfo.status of
+                    SyncManager.Model.NotAvailable ->
+                        showWarningMessage Translate.SelectedHCNotSynced Translate.PleaseSync
+
+                    SyncManager.Model.Downloading ->
+                        if syncInfo.remainingToDownload > 1500 then
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/Utils.elm:1286]: @anvmn What is this 1500? Seconds? Size? any reason for this particular number?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. One question to check.

File: client/src/elm/Pages/Utils.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/Utils.elm:1286]: Added comment

**Pull request merged**